aydin/nn/util/test/test_data_util.py

```diff
@@ -52,7 +53,9 @@ def test_random_sample_patch_2D():
     patch_size = (64, 64)
     num_patch = 500
     adoption_rate = 0.5
-    input_data = random_sample_patches(image0, patch_size, num_patch, adoption_rate)
+    input_data = legacy_random_sample_patches(
```

Comment:
It might be easier to understand if test names are different for legacy and new methods?